scss/detail.scss

```diff
@@ -78,6 +80,16 @@
   margin-left: 25px;
 }
 
+.book-navbar {
+  background-color: $white;
+  border-top: 5px solid $cerulean;
+  box-shadow: 0 4px 10px 0 $black-shadow;
+  display: flex;
+  position: fixed;
+  justify-content: space-around;
+  width: 1366px;
```

Comment:
Why the navbar have a fixed width?

Reply:
Remember that you have to thing on all devices